apollo-federation/src/sources/connect/expand/visitors/input.rs

```diff
@@ -58,17 +58,6 @@ impl FieldVisitor<InputObjectFieldDefinitionPosition>
 impl GroupVisitor<InputObjectTypeDefinitionPosition, InputObjectFieldDefinitionPosition>
     for SchemaVisitor<'_, InputObjectTypeDefinitionPosition, InputObjectType>
 {
-    fn get_group_fields(
-        &self,
-        group: InputObjectTypeDefinitionPosition,
-    ) -> Result<
-        Vec<InputObjectFieldDefinitionPosition>,
-        <Self as FieldVisitor<InputObjectFieldDefinitionPosition>>::Error,
-    > {
-        let def = group.get(self.original_schema.schema())?;
-        Ok(def.fields.keys().cloned().map(|f| group.field(f)).collect())
-    }
-
```

Comment:
Good call removing this. I originally had it to make it slightly more modular but then only used it in one spot that needed to be implemented anyway.